linkshell_service/linkshell_service.lua

```diff
@@ -0,0 +1,72 @@
+local packets = require('packets')
+local shared = require('shared')
+
+local linkshell = shared.new('linkshell')
+
+-- Does this file need this?
```

Comment:
It currently needs it, though it's debatable whether it *should* need it. Though I'd say that's for another time, so the comment is no longer necessary.

Reply:
https://github.com/Windower/packages/pull/63/commits/7d40d96e9dc89d3258969f887883bf12cf58c16a